Update/Update.fr.json

```diff
@@ -1,6 +1,6 @@
 {
   "CurrentVersionUpToDate": "Votre version actuelle ({0}) est la plus à jour !",
-  "DisableUpdateStartup": "Désactiver cette fenêtre au démarrage.",
+  "DisableUpdatePopupStartup": "Disable this popup at startup.",
```

Comment:
Why does it replace the french version with the english one?

Reply:
Because you introduced a new string and replaced/removed that one, to avoid mistranslation it's better to placeholder with the english one